config/env/development.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var defaultEnvConfig = require('./default');
+
+module.exports = {
+  db: {
+    uri: process.env.MONGOHQ_URL || process.env.MONGOLAB_URI || ('mongodb://admin:campusally_admin@ds149144.mlab.com:49144/campusally') + '/mean-dev', options: {
+      user: '',
+      pass: ''
+    },
+    // Enable mongoose debug mode
+    debug: process.env.MONGODB_DEBUG || false
+  },
+  log: {
+    // Can specify one of 'combined', 'common', 'dev', 'short', 'tiny'
+    format: 'dev',
+    // Stream defaults to process.stdout
+    // Uncomment to enable logging to a log on the file system
+    options: {
+      //stream: 'access.log'
+    }
+  },
+  app: {
+    title: defaultEnvConfig.app.title + ' - Development Environment'
+  },
+  facebook: {
+    clientID: process.env.FACEBOOK_ID || 'APP_ID',
+    clientSecret: process.env.FACEBOOK_SECRET || 'APP_SECRET',
+    callbackURL: '/api/auth/facebook/callback'
+  },
+  twitter: {
+    clientID: process.env.TWITTER_KEY || 'CONSUMER_KEY',
+    clientSecret: process.env.TWITTER_SECRET || 'CONSUMER_SECRET',
+    callbackURL: '/api/auth/twitter/callback'
+  },
+  google: {
+    clientID: process.env.GOOGLE_ID || 'APP_ID',
+    clientSecret: process.env.GOOGLE_SECRET || 'APP_SECRET',
+    callbackURL: '/api/auth/google/callback'
+  },
+  linkedin: {
+    clientID: process.env.LINKEDIN_ID || 'APP_ID',
+    clientSecret: process.env.LINKEDIN_SECRET || 'APP_SECRET',
+    callbackURL: '/api/auth/linkedin/callback'
+  },
+  github: {
+    clientID: process.env.GITHUB_ID || 'APP_ID',
+    clientSecret: process.env.GITHUB_SECRET || 'APP_SECRET',
+    callbackURL: '/api/auth/github/callback'
+  },
+  paypal: {
+    clientID: process.env.PAYPAL_ID || 'CLIENT_ID',
+    clientSecret: process.env.PAYPAL_SECRET || 'CLIENT_SECRET',
+    callbackURL: '/api/auth/paypal/callback',
+    sandbox: true
+  },
+  mailer: {
+from: process.env.MAILER_FROM || 'MAILER_FROM' || 'campusally1@gmail.com',
+options: {
+service: 'Gmail',
+secure: false,
+port:25,
+auth: {
+user: 'campusally1@gmail.com',
+pass: 'machosheep1'
+}
+}
+},
+  livereload: true
+};
```

Comment:
Is this file going to be uploaded to github ? or its included in gitignore ?

Reply:
It will go int git. Before we handle the app to Thor we will change the user and password of the mlab db and gmail for security reason; however, to be able to deploy we must have the file on git

Reply:
when I deployed to heroku, I did not put db password to github. I cant remember on top of my head right now but there is a way to avoid showing sensitive information to the public.